101-cosmosdb-webapp/README.md

```diff
@@ -0,0 +1,27 @@
+# Create an Azure Cosmos DB account and Web App and deploy web app from GitHub
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-cosmosdb-webapp/PublicLastTestDate.svg)
+![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-cosmosdb-webapp/PublicDeployment.svg)
+
+![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-cosmosdb-webapp/FairfaxLastTestDate.svg)
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-cosmosdb-webapp/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-cosmosdb-webapp/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-cosmosdb-webapp/CredScanResult.svg)
+
+This template creates an Azure Cosmos account and Azure Web App, then automatically deploys a [Sample To Do ASP MVC web app](https://github.com/Azure-Samples/cosmos-db-to-do) hosted on GitHub and injects the Cosmos DB endpoint and auth key into the Web App's Application Settings allowing it to connect automatically upon first run.
+
+This sample is useful where you want to deploy these resources and have the web app automatically connect to Cosmos DB in a single operation without having to manually add connection information to Application Settings in the portal.
+
+Below are the parameters which can be user configured in the parameters file including:
+
+- **Application Name:** Pick a name that will be used for the Cosmos account, App Plan and Web App.
+- **Location:** Azure region for all the resources.
+- **App Service Plan Tier:** App Service Plan's pricing tier.
+- **App Service Plan Instances:** App Service Plan's instance count.
+- **Repo URL:** The URL for the GitHub repository that contains the project to deploy.
+- **Branch:** The branch of the GitHub repository to use.
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-webapp%2Fazuredeploy.json)
+[![Deploy To Azure Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-webapp%2Fazuredeploy.json)
```

Comment:
fix gov button

Reply:
I can't see what is wrong with gov cloud button. Can you point to it?

Reply:
found it. fixed in next commit